core/domain/exp_domain.py

```diff
@@ -6575,7 +6573,7 @@ def __init__(
         self.param_specs = param_specs
         self.param_changes = param_changes
         self.auto_tts_enabled = auto_tts_enabled
-        self.correctness_feedback_enabled = correctness_feedback_enabled
+        self.correctness_feedback_enabled = True
```

Comment:
Can we just drop the field?

Reply:
I did drop this entirely in an old commit, but to restore CMD, I had to bring it back. We are not directly updating it; instead, it still persists in the backend only for CMD purposes and testing. It is not being used anywhere else, just like STATE_PROPERTY_WRITTEN_TRANSLATIONS_DEPRECATED and STATE_PROPERTY_NEXT_CONTENT_ID_INDEX_DEPRECATED.

Reply:
Why do you need this for the CMD? Can you explain? 

It should actually be removed entirely (conceptually) so I want to understand the disconnect here.

Reply:
I followed the approach used for next_content_id (which is not applicable for this case). here, instead of writing a separate function just for updating the correctness_feedback_enabled property in the datastore field, we instead update the domain object and save it. this is the only thing i'm confused about. can you give some hints on how should i do this without the domain object?

Reply:
dropped the CMD as suggested below.